confgenerator/testdata/invalid/linux/combined-receiver_otlp_missing_traces/golden/error

```diff
@@ -0,0 +1 @@
+combined receiver "otlp" found with no traces section; did you forget to define a traces pipeline?
```

Comment:
Can we make the action item more explicit in this error message?

e.g. `when an "otlp" type receiver is specified in the "combined" section, two pipelines are required separately in the metrics and trace sections to include this receiver.`

Reply:
Improved, PTAL.